ItemList/ItemList/ViewModel.cs

```diff
@@ -0,0 +1,146 @@
+        /// Stores an instance of RelayCommand to delete.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        private RelayCommand deleteCommand;
+        /// <summary>
+        /// Stores instance of selected Phone.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        public Phone selectedItem;
+        /// <summary>
+        /// Initializes addCommand field.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        /// <returns>Instance of RelayCommand class.</returns>
+        public RelayCommand AddCommand
+        {
+            get
+            {
+                if (this.addCommand != null)
+                {
+                    return this.addCommand;
+                }
+                this.addCommand = AddNewItemToCollection();
+                return this.addCommand;
+            }
+        }
+        /// <summary>
+        /// Command adds new item to Observable Collection of phones and clean text boxes.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        /// <returns>Instance of RelayCommand class.</returns>
+        private RelayCommand AddNewItemToCollection ()
+        {
+            return new RelayCommand(obj =>
+            {
+                this.ListOfItems.Add(new Phone { Manufacture = this.Manufacture, Model = this.Model });
+                ClearField();
+                UpdateTextBoxes();
+            });
+        }
+        /// <summary>
+        /// Method is used for clearing text boxes.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        private void ClearField()
+        {
+            Manufacture = string.Empty;
+            Model = string.Empty;
+        }
+        /// <summary>
+        /// Calls OnPropertyChanged event
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        private void UpdateTextBoxes()
+        {
+            OnPropertyChanged("Manufacture");
+            OnPropertyChanged("Model");
+        }
+        /// <summary>
+        /// Gets delete command for collection of phones.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        /// <returns>Instance of RelayCommand class.</returns>
+        public RelayCommand DeleteCommand
+        {
+            get
+            {
+                return this.deleteCommand ??
+                    (this.deleteCommand = new RelayCommand(obj =>
+                    {
+                        this.ListOfItems.Remove(this.selectedItem);
+                    }));
+            }
+        }
+        /// <summary>
+        /// Gets or sets collection of items.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        public ObservableCollection<Phone> ListOfItems { get; set; }
+        /// <summary>
+        /// Gets or sets manufacture of the item.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        /// <returns>Manufacture name as a string.</returns>
```

Comment:
In return part it is not necessary to specify a type.
"Manufacture name of the phone."